packages/slider/src/slider.ts

```diff
@@ -74,6 +75,12 @@ export class Slider extends Focusable {
     /* Ensure that a '' value for `variant` removes the attribute instead of a blank value */
     private _variant = '';
 
+    @property({ attribute: false })
+    public getAriaValueText: (value: number) => string = (value) => `${value}`;
```

Comment:
TS covers us here, I guess, but do we want to do anything about this getting set badly? A fallback to default or something?

Reply:
Added a basic guard in the new getter version of `ariaValueText` which if `getAriaValueText` is falsy will use the default string. I don't think we need to handle all cases I think, just enough to allow someone to clear their changes is good?